WebAPIAssignment/AirProduct.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace WebAPIAssignment
+{
+    class AirProduct : IProduct
+    {
+        bool bookingStatus = false;
+        int fare = 100;
+
+        public string GetTypeOfProduct()
+        {
+            Logger log = Logger.GetLogger();
+            log.WriteMessage("GetTypeOfProduct function called");
+            return "Air";
+        }
+        public void Save(string name, int choice)
+        {
+            AirFare obj = new AirFare();
+            fare = obj.FinalFare(fare);
+            Logger log = Logger.GetLogger();
+            log.WriteMessage("Save function called");
+            Console.WriteLine("Saving air product");
+            if (choice == 1)
+            {
+                LocalFile file = new LocalFile();
```

Comment:
instead of calling like this we cold have passed the interface object in the method itself so it doesn't violate the solid principle later